### DISCUSSION TITLE
[FEATURE]方便的进行端口映射, 容器允许修改127.0.0.1默认设置

### DISCUSSION THREAD

**Discussion by 1057105012** [1Panel-dev/1Panel#7449] [Q&A]
### 1Panel 版本

方便的进行端口映射, 容器允许修改127.0.0.1默认设置

### 请描述您的需求或者改进建议

容器默认127.0.0.1, 都没法改, 而且, nginx不支持直接映射到本地或者远程服务器端口做转发, 还必须填写文件夹之类的, 比如我要转发docker, 就没法弄

### 请描述你建议的实现方案

1. 容器支持更改端口映射, 最好别重启容器, 容易数据丢失
2. openresty和nginx提供端口映射, 不要填写那么多东西, 没有方便的端口转发功能, 容器127.0.0.1绑定, nginx又集成的太厉害, 没法改, 直接给绑死了

**Comment by zhengkunwang223** [comment]
1. 改端口映射必须重启容器，容器重启数据丢失是因为没有把数据目录挂载出来
2. 再详细描述一下 你想要的端口映射是什么 我们当前的 Openresty 是 Host 模式 端口用的是宿主机端口

**Comment by QYG2297248353** [comment]
同时建议： 一键部署 可以切换绑定应用

**Comment by wanghe-fit2cloud** [comment]
将该 Issue 转为 Discussion 了
